cmd/subnetcmd/deploy.go

```diff
@@ -104,6 +104,17 @@ func deploySubnet(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	chain := chains[0]
+
+	sc, err := app.LoadSidecar(chain)
+	if err != nil {
+		return fmt.Errorf("failed to load sidecar for later update: %w", err)
+	}
+
+	if sc.ImportedFromAPM {
```

Comment:
why is this restriction?

Reply:
Because the genesis data is not included in the apm repo. It's possible we could retrieve it from the running network, but that would be an extension.

Reply:
I think we may need a way to provide a genesis when importing the subnet, or maybe add it later, either by modifying the subnet config with a new command, or by adding an option to deploy.

Reply:
maybe we want a new issue for that

Reply:
We can try addressing in the next release.

https://github.com/ava-labs/avalanche-cli/issues/238